Sources/OpenAI/Supporting Types/Safety.swift

```diff
@@ -0,0 +1,61 @@
+/**
+ The result of a contentFilter request.
+ */
+public enum Safety: Int {
+    
+    /// This means that the text has been evaluated as safe.
+    case safe = 0
+    
+    /// This means that the text could be talking about a sensitive topic, something political, religious, or talking about a protected class such as race or nationality.
+    case sensitive = 1
+    
+    /// This means that the text contains profane language, prejudiced or hateful language, something that could be NSFW, or text that portrays certain groups/people in a harmful manner.
+    case unsafe = 2
+    
+    /// This means that the completion request return an unknown token and has failed.
+    case failure = 3
```

Comment:
A better way to represent a failure case is either a `nil` value of `Safety?` or (preferably) a thrown error or a `.failure` case for a `Result` value. 

```suggestion
```